### ISSUE TITLE
How to use react-cool-virtual on window? (Not on scroll containers like in the examples)

### ISSUE THREAD

**Issue by wansaleh** [wellyshen/react-cool-virtual#144]
Hi. Thanks for this awesome virtualization library.

However I couldn't found an example detailing usage on a window. Something like the [windowscroller](https://github.com/bvaughn/react-virtualized/blob/master/docs/WindowScroller.md) in react-virtualized.
Can you explain/provide example on how to use react-cool-virtual on a window instead of a scroll container with an `outerRef`?

Best regards.

**Comment by wellyshen** [comment]
@wansaleh I'm sorry this hook hasn't supported the feature yet. I will let you know when it's supported.

**Comment by wansaleh** [comment]
Thank you so much for the clarification!

**Comment by matyas-igor** [comment]
Agree that library is amazing! Would also love to have it worked with window or other element as a scroll container

**Comment by henrybabbage** [comment]
Agreed that a window scrolling feature would be an awesome enhancement to this already really great library.
